stdlib/public/core/Unicode.swift

```diff
@@ -645,22 +645,20 @@ public func transcode<
 
   var inputDecoder = inputEncoding.init()
```

Comment:
Should we use `InputEncoding()` instead of `inputEncoding.init()` or `inputEncoding()`?